crud/error.go

```diff
@@ -0,0 +1,79 @@
+package crud
+
+import (
+	"fmt"
+)
+
+// CrudError describes CRUD error object.
```

Comment:
Is this a tarantool/errors class mapper?

Reply:
It is mapper for crud error.